backend/src/main/java/com/onwelo/practice/bts/entity/BankAccount.java

```diff
@@ -1,15 +1,18 @@
 package com.onwelo.practice.bts.entity;
 
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
 import javax.persistence.*;
-import java.util.Collection;
+import java.util.List;
 
 @Entity
-@Table(name = "bank_account")
 public class BankAccount {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(unique = true)
     private String bankNumber;
```

Comment:
Nie wiem czy nie lepiej brzmiałoby accNumber/accountNo. Nazwa bankNumber sugeruje że to jest jakiś numer banku a nie konta.